cmd/juju/user/change_password.go

```diff
@@ -27,16 +28,27 @@ const userChangePasswordDoc = `
 The user is, by default, the current user. The latter can be confirmed with
 the ` + "`juju show-user`" + ` command.
 
-A controller administrator can change the password for another user (on
-that controller).
+A controller administrator can change the password for another user 
+on the specified controller. If no controller is specified, current controller
+will be used.
+
+A controller administrator can also reset the password for another user.
```

Comment:
The sentence is not the same :D 
First one says "..can change...", Second one says ".. can also reset.."